sdk/communication/communication-phone-numbers/review/communication-phone-numbers.api.md

```diff
@@ -271,20 +303,34 @@ export interface SipRoutingError {
     readonly target?: string;
 }
 
+// @public
+export interface SipRoutingGetOptionalParams extends coreClient.OperationOptions {
+    expand?: ExpandEnum;
```

Comment:
Please use this also in the getTrunks, so customers can retrieve extended health info on for one trunk.